BUILD

```diff
@@ -22,6 +22,7 @@ exports_files(["LICENSE"])
 
 strict_warnings_objc_library(
     name = "MDFTextAccessibility",
+    module_name = "MDFTextAccessibility",
```

Comment:
This resolves a swift compiler error in the unit tests that the module name couldn't be found.